src/mwoperators/DerivativeKernel.h

```diff
@@ -21,6 +21,8 @@ class DerivativeKernel : public GreensKernel {
         GaussPoly<1> dg = g.differentiate(0);
         this->append(dg);
     }
+
+    virtual std::ostream& print(std::ostream &o) const { return o; }
```

Comment:
Isn't it better to just overload `operator<<` instead of passing a stream and returning it?

Reply:
I took this from your coding standards :stuck_out_tongue: `operator<<` is overloaded in the base class which calls a virtual `print()` function that is implemented in the derived classes. Here I just added a quiet print since it was a missing pure virtual and Luca's compiler complained (not mine for some reason...).

Reply:
LOL. Can you add a comment that this is a silent print? The implementation is a bit confusing :) BTW what was the compiler warning/error?

Reply:
Implemented a proper print instead